compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -0,0 +1,308 @@
+package dotty
+package tools
+package dotc
+
+import org.junit.Test
+import java.io.{ File => JFile }
+
+import org.junit.experimental.categories.Category
+
+@Category(Array(classOf[ParallelTesting]))
+class CompilationTests extends ParallelTesting {
+  import CompilationTests._
+
+  def interactive: Boolean = !sys.env.contains("DRONE")
+
+  def regex: Option[String] = sys.props.get("dotty.partest.filter")
+
+  // Positive tests ------------------------------------------------------------
+
+  @Test def compilePos: Unit = {
+    compileList("compileStdLib", StdLibSources.whitelisted, scala2Mode.and("-migration", "-Yno-inline")) +
+    compileFilesInDir("../tests/pos", defaultOptions)
+  }.pos()
```

Comment:
I didn't mean to drop `()`, I've meant to give it another name that indicate that it's going to actually run the test. It would be also nice to make it not clash with `pos`ition.